lib/config/index.js

```diff
@@ -1298,14 +1298,21 @@ Config.prototype._canonicalize = function _canonicalize() {
   const level = this.logging.level
   this.logging.level = logAliases[level] || level
 
+  const region = parseKey(this.license_key)
   if (this.host === '') {
-    const region = parseKey(this.license_key)
     if (region) {
-      this.host = 'collector.' + region + '.nr-data.net'
+      this.host = `collector.${region}.nr-data.net`
     } else {
       this.host = 'collector.newrelic.com'
     }
   }
+  if (this.otlp_endpoint === '') {
```

Comment:
cool so to point this to staging we'd have to set `NEW_RELIC_OTLP_ENDPOINT`?

Reply:
Yeah, just like `NEW_RELIC_HOST`.